solutions/instances/main.tf

```diff
@@ -149,7 +149,44 @@ module "scc" {
 }
 
 #######################################################################################################################
-# SCC WP
+# SCC Attachment
+#######################################################################################################################
+
+# Data source to account settings
+data "ibm_iam_account_settings" "iam_account_settings" {}
+
+module "create_profile_attachment" {
+  source  = "terraform-ibm-modules/scc/ibm//modules/attachment"
+  version = "1.5.0"
+  for_each = {
+    for idx, profile_attachment in var.profile_attachments :
+    profile_attachment => idx
+  }
+  profile_name           = each.key
+  profile_version        = "latest"
+  scc_instance_id        = module.scc.guid
+  attachment_name        = "${each.value + 1} daily full account attachment"
+  attachment_description = "SCC profile attachment scoped to your specific IBM Cloud account id ${data.ibm_iam_account_settings.iam_account_settings.account_id} with a `daily` attachment schedule."
```

Comment:
remove the backticks from the description - I dont think it will render in SCC UI